test/unit/util.test.ts

```diff
@@ -65,3 +67,45 @@ test('Get full URL', async(t) => {
             'https://wikimedia.org/api/rest_v1/media/math/render/svg/34cbb1e27dae0c04fc794a91f2aa001aca7054c1',
             'Full Url when base and url both strtas with http/s');
 })
+
+test('Custom flavour path', async(t) => {
+
+    // checks in current working directory.
+    let createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'testCustomFlavour.js'));
+
+    t.equal(getCustomFlavorPath('testCustomFlavour.js'), pathParser.resolve(process.cwd(), 'testCustomFlavour.js'),
+        'Custom flavour in working directory.');
+
+    // checks when current and working directory have file with same name(preference to working directory).
+    createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'extensions/testCustomFlavour.js'));
```

Comment:
Same here, but moving one directory level higher.

Reply:
yes here I will move directly instead of moving from current working directory